pandas/tests/frame/methods/test_astype.py

```diff
@@ -565,3 +566,18 @@ def test_astype_empty_dtype_dict(self):
         result = df.astype(dict())
         tm.assert_frame_equal(result, df)
         assert result is not df
+
```

Comment:
do we have tests for these same types for errors='raise' (the default) and errors='coerce'?

Reply:
I don't think so for all, can add those here